### PR TITLE
proximity_list: correct the name of the place object by using fill_pb_placemark

### PR DESCRIPTION
**Pull request by TeXitoi**

### CODE REVIEW

**Comment by antoine-de** [comment]
can you add tests ? :smile: 

jormun integration test could be nice on places/id

**Comment by TeXitoi** [comment]
JIRA: http://jira.canaltp.fr/browse/NAVITIAII-1112

File: source/jormungandr/tests/places_test.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+@dataset(["main_routing_test"])
+class TestPlaces(AbstractTestFixture):
+    """
+    Test places responses
+    """
+
+    def test_places_by_id(self):
+        """can we get the complete address from coords"""
+
+        lon = 10. / 111319.9
```
Comment (antoine-de) [inline source/jormungandr/tests/places_test.py:45]: can you put a comment about this magic number ?

File: source/jormungandr/tests/check_utils.py [hunk 2]
```diff
@@ -547,12 +547,40 @@ def is_valid_line(line, depth_check=1):
...
+    else:
+        assert(False, "invalid type")
+
+
+def is_valid_address(address, depth_check=1):
+    get_not_null(address, "id")
+    get_not_null(address, "house_number")
+    get_not_null(address, "name")
+    get_not_null(address, "administrative_regions") # TODO test
+    get_not_null(address, "coord") # TODO test
```
Comment (antoine-de) [inline source/jormungandr/tests/check_utils.py]: you can use is_valid_{lon,lat} for the coords (maybe it's better to embed that in a is_valid_coord)

**Pull request merged**